lib/resources/styles.css

```diff
@@ -79,10 +79,6 @@ main {
   padding: 20px 15px 0 30px;
 }
 
-.main-content p {
```

Comment:
are there any consequences to removing this style?

Reply:
I believe this just applies the max-width property, which is now applied using the `class="markdown desc"` selector below.